lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -4204,6 +4240,14 @@ ParseResult FIRCircuitParser::parseCircuit(
       emitError("unexpected token in circuit");
       return failure();
 
+    case FIRToken::kw_type: {
+      if (FIRVersion::compare(version, FIRVersion({3, 0, 0})) < 0)
```

Comment:
3.0.0 is insufficient for this, but I understand there's no version with this contained yet so hard to say what this /should/ be yet.  Anyway, I get why its useful to support parsing before its in the spec, just don't want to be looking at this in X months wondering why we check for "wrong" version here.